EiffelArtifactDeployedEventV0_1_0.go

```diff
@@ -0,0 +1,137 @@
+// Copyright Axis Communications AB.
+//
+// For a full list of individual contributors, please see the commit history.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// THIS FILE IS AUTOMATICALLY GENERATED AND MUST NOT BE EDITED BY HAND.
+
+package eiffelevents
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/clarketm/json"
+	"github.com/google/uuid"
+)
+
+// NewArtifactDeployedV0_1_0 creates a new struct pointer that represents
+// major version 0 of EiffelArtifactDeployedEvent.
+// The returned struct has all required meta members populated.
+// The event version is set to the most recent 0.x.x
+// currently known by this SDK.
```

Comment:
Small nit: Does this description work in this case? I mean, the function name "NewArtifactDeployedV0_1_0" and the line "is set to the most recent 0.x.x" seems to contradict each other a bit.

Reply:
Ah, right. Good catch. I added a conditional to render different texts for v0 events.